docs/authoring.md

```diff
@@ -17,11 +17,12 @@ The AsyncAPI generator has been built with extensibility in mind. The package us
-   - When using `$$channel$$` you have access to two variables `channel` and `channelName`. Where `channel` contains the current channel being rendered.
-   - When using `$$message$$` you have access to two variables `message` and `messageName`. Where `message` contains the current message being rendered.
-   - When using `$$schema$$` you have access to two variables `schema` and `schemaName`. Where `schema` contains the current schema being rendered.
-   - When using `$$parameter$$` you have access to two variables `parameter` and `parameterName`. Where `parameter` contains the current parameter being rendered.
-   - When using `$$securityScheme$$` you have access to two variables `securityScheme` and `securitySchemeName`. Where `securityScheme` contains the current security scheme being rendered.
+   - When using `$$channel$$` you have access to two variables [`channel`](https://github.com/asyncapi/parser-js/blob/master/API.md#Channel) and [`channelName`](https://github.com/asyncapi/parser-js/blob/master/API.md#AsyncAPIDocument+channels). Where `channel` contains the current channel being rendered.
+   - When using `$$message$$` you have access to two variables [`message`](https://github.com/asyncapi/parser-js/blob/master/API.md#Message) and [`messageName`](https://github.com/asyncapi/parser-js/blob/master/API.md#Message+uid). Where `message` contains the current message being rendered.
+   - When using `$$schema$$` you have access to two variables [`schema`](https://github.com/asyncapi/parser-js/blob/master/API.md#Schema) and [`schemaName`](https://github.com/asyncapi/parser-js/blob/master/API.md#Schema+uid). Where `schema` contains the current schema being rendered. Only schemas from [Components object](https://www.asyncapi.com/docs/specifications/2.0.0/#a-name-componentsobject-a-components-object) are used. 
+   - When using `$$everySchema$$` you have access to two variables [`schema`](https://github.com/asyncapi/parser-js/blob/master/API.md#Schema) and [`schemaName`](https://github.com/asyncapi/parser-js/blob/master/API.md#Schema+uid). Where `schema` contains the current schema being rendered. Every [Schema object](https://www.asyncapi.com/docs/specifications/2.0.0/#schemaObject) from the entire AsyncAPI file is used.
+   - When using `$$parameter$$` you have access to two variables [`parameter`](https://github.com/asyncapi/parser-js/blob/master/API.md#ChannelParameter) and [`parameterName`](https://github.com/asyncapi/parser-js/blob/master/API.md#Channel+parameters). Where `parameter` contains the current parameter being rendered.
+   - When using `$$securityScheme$$` you have access to two variables [`securityScheme`](https://github.com/asyncapi/parser-js/blob/master/API.md#SecurityScheme) and [`securitySchemeName`](https://github.com/asyncapi/parser-js/blob/master/API.md#Components+securitySchemes). Where `securityScheme` contains the current security scheme being rendered.
```

Comment:
Further more for securityScheme and channel, when the name links me to the https://github.com/asyncapi/parser-js/blob/master/API.md#Components+securitySchemes and https://github.com/asyncapi/parser-js/blob/master/API.md#AsyncAPIDocument+channels I would expect I would have access to the map instead of just the key (which is what is provided) maybe add some extra text for those 🤔 ?

Reply:
I understand your concern, but to the pity we do not have a direct way in API to get only the name of channel, channel parameter or security schema. We had them only in the list of Object.<string, Type>.
For me there is a thread of between simplicity, compactness of explanation and ton of hints in bracers.
If you have some good sentence with explanation in mind, please suggest.

Reply:
Left a new review with suggestion 😄